.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+name: 'Release'
+on:
+  release:
+    # runs for stable and pre-releases
+    types: [published]
+
+jobs:
+  cancel-previous-workflow:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@b173b6ec0100793626c2d9e6b90435061f4fc3e5 # pin@0.11.0
+        with:
+          access_token: ${{ github.token }}
+
+  release:
+    name: Release Job ${{ matrix.os }} - Java ${{ matrix.java }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        java: ['17']
+    env:
+      OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+      OSSRH_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
+      GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
+      GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v4
+
+      - name: 'Set up Java: ${{ matrix.java }}'
+        uses: actions/setup-java@v3
+        with:
+          java-version: ${{ matrix.java }}
+          distribution: 'temurin'
+
+      - name: Update version
+        run: ./scripts/bump-version.sh ${{ github.event.release.tag_name }}
+
+      - name: Commit & push
+        run: |
+          echo "Tag name from github.ref_name: ${{ github.event.release.target_commitish }}"
+          ./scripts/commit-code.sh ${{ github.event.release.target_commitish }}
+
+      # Let's be sure that everything works before we release    
+      - name: Dry release
+        run: make dryRelease
+  
+      # - name: Release
+      #   run: make release
```

Comment:
Remove comments when publishing the 1st alpha, this will upload artifacts to sonatype.